diff --git a/include/LIEF/runtime/osx/Host.hpp b/include/LIEF/runtime/osx/Host.hpp
index ec38cc5..644f276 100644
--- a/include/LIEF/runtime/osx/Host.hpp
+++ b/include/LIEF/runtime/osx/Host.hpp
@@ -31,9 +31,9 @@ class LIEF_API Host {
     uint32_t patch = 0;
 
     version_t(uint32_t major, uint32_t minor, uint32_t patch) :
-      major(major),
-      minor(minor),
-      patch(patch) {}
+      major{major},
+      minor{minor},
+      patch{patch} {}
 
     bool operator<=(const version_t& rhs) const;
     bool operator>(const version_t& rhs) const {
diff --git a/include/LIEF/runtime/windows/Host.hpp b/include/LIEF/runtime/windows/Host.hpp
index 3f508d1..d0f6bb7 100644
--- a/include/LIEF/runtime/windows/Host.hpp
+++ b/include/LIEF/runtime/windows/Host.hpp
@@ -34,9 +34,9 @@ class LIEF_API Host {
     version_t() = default;
 
     version_t(uint32_t major, uint32_t minor, uint32_t build_number) :
-      major(major),
-      minor(minor),
-      build_number(build_number) {}
+      major{major},
+      minor{minor},
+      build_number{build_number} {}
 
     bool operator<=(const version_t& rhs) const;
     bool operator>(const version_t& rhs) const {
